testing_and_setup/compass/ocean/scripts/vertical_grid/define_vertical_grid_64_layers.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+import subprocess
+
+# Standard 64 layer vertical grid
+subprocess.check_call(['./make_vertical_grid.py', 
+    '--output_file_name', 'vertical_grid.nc',
+    '--num_vert_levels', '64',
+    '--bottom_depth', '5500',
+    '--layer1_thickness', '2',
+    '--maxLayer_thickness', '200',
+    '--plot_vertical_grid'])
```

Comment:
These kinds of subprocess calls are not working well with the MPI versions of the COMPASS environment.  It would be better to make this a function and call it that way.  In general, calling python scripts from other python scripts with subprocess is not a good practice.

Reply:
@mark-petersen, do you want me to make these changes or do you want to do it?

Reply:
So it should look like this instead?
```
include make_vertical_grid

make_vertical_grid(arg1, arg2, ...)
```

Reply:
OK, that one is done.

Reply:
It should look like this:
```python
#!/usr/bin/env python
from make_vertical_grid import create_vertical_grid


# Standard 64 layer vertical grid
create_vertical_grid(bottom_depth=5500, nz=64, dz1_in=2.0, dz2_in=200.0,
                     plot_vertical_grid=True, maxit=1000, epsilon=1e-2,
                     outfile='vertical_grid.nc')
```